amxmodx/CLang.h

```diff
@@ -118,7 +113,7 @@ class CLangMngr
 	
 	LangVec m_Languages;
 
-	CVector<md5Pair *> FileList;
+	StringHashMap<time_t> FileList;
 	CVector<String *> KeyList;
 	THash<String, keytbl_val> KeyTable;
```

Comment:
Exactly. xPaw, I'm not familiarized with amtl. Really, I'd appreciate Arkshine's hands doing these amtl implementations. I will help any other ways, if needed, for now.

Agree, that CLang code really needed a change.